README.md

```diff
@@ -203,14 +204,46 @@ Space      | Skip cutscenes, interact
 Ctrl + c   | Force Quit (from command-line)
 Ctrl + q   | Quit (ingame)
 
+## 5. Running The Longest Journey
+
+### 5.1. Required files ###
+
+You will need these elements from your game CDs or DVD :
+  * *TODO*
+  
+For digital distributions (Steam, GOG, demo from Steam or other sources),
```

Comment:
For the Steam version at least, the font files are missing. They need to be obtained by some other way. For example from the demo version.

Reply:
I noticed the fonts are missing from both the Steam version and *Steam* demo (gotta check the demo from other sources) – the game works without them but the fancy fonts are indeed missing and ResidualVM mentions that, so guess it's worth adding. On the other hand it seems that Steam version is pre-patched with [patch 1.61](https://community.pcgamingwiki.com/files/file/345-the-longest-journey-patch/), but I'm not sure if there's any files that are related to ResidualVM as after extracting the EXE I can only see configurator tool and updated dll files inside, other than maybe some ini updates. Steam manual is updated for retail versions as well (mostly didn't mention CD or DVD requirements). On the other hand GOG include a Police Station fix, but I'm not sure if the version differences/patches are to be mention in this documentation?

Reply:
I usually use this demo version to get the fonts: https://archive.org/details/Longest_Journey_demo.

ResidualVM doesn't need any of the patches. I don't think it's needed to mention them.